src/sqlitetablemodel.cpp

```diff
@@ -432,6 +432,26 @@ bool SqliteTableModel::removeRows(int row, int count, const QModelIndex& parent)
     return true;
 }
 
+QModelIndex SqliteTableModel::dittoRecord(int old_row)
+{
+    int firstEditedColumn = 0;
+    int new_row = rowCount() - 1;
+
+    sqlb::Table t = sqlb::Table::parseSQL(m_db->getObjectByName(m_sTable).getsql()).first;
+    for (int col = 0; col < t.fields().size(); ++col) {
+        if (!t.fields().at(col)->primaryKey()) {
+            if (!firstEditedColumn)
+                firstEditedColumn = col + 1;
+
+            QVariant value = m_data.at(old_row).value(col + 1);
+            if (!value.isNull())
```

Comment:
I did use that, but then replaced, cuz it copied NULL as text.
I ll send final patch set tomorrow. Btw, feel free to assign me on issues.